examples/advanced/edge/jobs/cifar10_sync_tp_runner/app/config/tp_config.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Change to simulation_config.json so you can use the tree_prov generated package easily.